server/database/migrations/20190805160738-create-user-follower.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
To be consistent, could you please use ES6 syntax here and in the other migration files?

Reply:
Thank you